0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -204,29 +204,32 @@
 # In[14]:
 
 
+micdfs = micdfs.loc[:, ~micdfs.columns.str.contains("__antehoc_group1")]
+micdfs.columns = micdfs.columns.str.replace("__antehoc_group0", "", regex=False)
```

Comment:
Consider a short comment for why you are removing the antehoc group string.

Maybe something like: 
`# Remove duplicate antehoc columns`

Reply:
I think they should know there are 2 columns for every antehoc and posthoc column, since these are pairwise comparisons, but I'll just mention it here